installing/install_config/customizations.adoc

```diff
@@ -5,91 +5,175 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 You complete most of the cluster configuration and customization after you
-deploy your {product-title} cluster. By default, a number of
+deploy your {product-title} cluster. A number of
 _configuration resources_
 //and _Custom Resources_
 are available.
 
-You modify the configuration resources to define the major components of the
-cluster, such as the networking configuration and the identity provider.
+You modify the configuration resources to configure the major components of the
```

Comment:
minor question: components or features? I think of them as features.

Reply:
i think of them as components (the bits of code implementing the feature) but you're probably right that to a user it's a feature.  i'll change it.